store/bolt/user_store.go

```diff
@@ -0,0 +1,124 @@
+package bolt
+
+import (
+	"errors"
+
+	"github.com/boltdb/bolt"
+	"github.com/gopheracademy/congo"
+	"github.com/gopheracademy/congo/store/bolt/internal"
+)
+
+var (
+	// ErrUserNotFound is returned when mutating a user that doesn't exist.
+	ErrUserNotFound = errors.New("user not found")
+
+	// ErrEmailRequired is returned when creating a user without an email.
+	ErrEmailRequired = errors.New("email address required")
+)
```

Comment:
This is a list of error conditions that can occur in the `UserStore`. I personally like a well-defined error list because it forces me to expose exactly how the code can fail. It also makes testing error conditions easier since I can check the returned error with the equality operator (`==`) instead of comparing the string message of the error.